cmd/loop/main.go

```diff
@@ -136,53 +136,63 @@ func getMaxRoutingFee(amt btcutil.Amount) btcutil.Amount {
 	return swap.CalcFee(amt, maxRoutingFeeBase, maxRoutingFeeRate)
 }
 
-type limits struct {
-	maxSwapRoutingFee   *btcutil.Amount
-	maxPrepayRoutingFee *btcutil.Amount
+type inLimits struct {
+	maxMinerFee btcutil.Amount
+	maxSwapFee  btcutil.Amount
+}
+
+func getInLimits(quote *looprpc.InQuoteResponse) *inLimits {
+	return &inLimits{
+		// Apply a multiplier to the estimated miner fee, to not get
+		// the swap canceled because fees increased in the mean time.
+		maxMinerFee: btcutil.Amount(quote.MinerFee) * 3,
+		maxSwapFee:  btcutil.Amount(quote.SwapFee),
+	}
+}
+
+type outLimits struct {
+	maxSwapRoutingFee   btcutil.Amount
+	maxPrepayRoutingFee btcutil.Amount
 	maxMinerFee         btcutil.Amount
 	maxSwapFee          btcutil.Amount
-	maxPrepayAmt        *btcutil.Amount
+	maxPrepayAmt        btcutil.Amount
 }
 
-func getLimits(amt btcutil.Amount, quote *looprpc.QuoteResponse) *limits {
+func getOutLimits(amt btcutil.Amount,
+	quote *looprpc.OutQuoteResponse) *outLimits {
+
 	maxSwapRoutingFee := getMaxRoutingFee(amt)
 	maxPrepayRoutingFee := getMaxRoutingFee(btcutil.Amount(
 		quote.PrepayAmt,
 	))
 	maxPrepayAmt := btcutil.Amount(quote.PrepayAmt)
 
-	return &limits{
-		maxSwapRoutingFee:   &maxSwapRoutingFee,
-		maxPrepayRoutingFee: &maxPrepayRoutingFee,
+	return &outLimits{
+		maxSwapRoutingFee:   maxSwapRoutingFee,
+		maxPrepayRoutingFee: maxPrepayRoutingFee,
 
 		// Apply a multiplier to the estimated miner fee, to not get
 		// the swap canceled because fees increased in the mean time.
 		maxMinerFee: btcutil.Amount(quote.MinerFee) * 100,
 
 		maxSwapFee:   btcutil.Amount(quote.SwapFee),
-		maxPrepayAmt: &maxPrepayAmt,
+		maxPrepayAmt: maxPrepayAmt,
 	}
 }
 
-func displayLimits(swapType swap.Type, amt, minerFees btcutil.Amount, l *limits,
-	externalHtlc bool, warning string) error {
+func displayInLimits(amt, minerFees btcutil.Amount,
```

Comment:
Do we need the warning string here, looks like we don't use it for loop in?

Reply:
Removed. This unified concept has been going on for too long already.